gocd/pipelines/snuba-stable.yaml

```diff
@@ -0,0 +1,109 @@
+# More information on gocd-flavor YAML can be found here:
+# - https://github.com/tomzo/gocd-yaml-config-plugin#pipeline
+# - https://www.notion.so/sentry/GoCD-New-Service-Quickstart-6d8db7a6964049b3b0e78b8a4b52e25d
+format_version: 10
+pipelines:
+    deploy-snuba-stable:
+        environment_variables:
+            GCP_PROJECT: internal-sentry
+            GKE_CLUSTER: zdpwkxst
+            GKE_REGION: us-central1
+            GKE_CLUSTER_ZONE: b
+            GKE_BASTION_ZONE: b
+            # Required for checkruns.
+            GITHUB_TOKEN: "{{SECRET:[devinfra-github][token]}}"
+        group: snuba
+        lock_behavior: unlockWhenFinished
+        materials:
+            snuba_repo:
+                git: git@github.com:getsentry/snuba.git
+                shallow_clone: true
+                branch: master
+                destination: snuba
+        stages:
+            - checks:
+                  approval:
+                      type: manual
+                  fetch_materials: true
+                  jobs:
+                      checks:
+                          timeout: 1800
+                          elastic_profile_id: snuba
+                          tasks:
+                              - script: |
+                                    /devinfra/scripts/checks/githubactions/checkruns.py \
```

Comment:
Where does this script live? Does it change how these checks are shown in Github?

Reply:
Also, why are these checks part of the deployment pipeline? They get run automatically on the master branch whenever we merge. Does this mean they all need to be run again before we can deploy?

Reply:
https://github.com/getsentry/devinfra-deployment-service/tree/main/gocd_agent/scripts/checks/githubactions

This script checks for the statuses, it doesn't run them.